### ISSUE TITLE
test external binary

### ISSUE THREAD

**Issue by andrewning** [byuflowlab/Minpack.jl#1]
@jennewcomb178 Here is an updated version of Minpack for your testing.  It's basically the same as the old one, but I updated the syntax since some of the calls were deprecated with the latest Julia.
@taylormcd Can you test whether this works for you? (it has an external fortran binary)  This is simpler than snopt so I figured let's start with this.  Be sure to actually install it (Pkg.clone() or clone it in a path that is in your LOAD_PATH)

**Comment by andrewning** [comment]
Ok, I made some updates.  There is a makefile so you shouldn't need to do the compilation command on your own. It also detects if you are on linux or a mac.  I also fixed the relative file issue @taylormcd mentioned with snopt. Give it a try and let me know if it works for you.

Should be able to Pkg.clone("...") then Pkg.build("Minpack")

**Comment by jennewcomb178** [comment]
The package tests worked for me and it worked first try implemented in my derivative module. However, I did get the same errors from ifort from the build.

**Comment by andrewning** [comment]
Yeah I know.  Mine at least shows the gfortran compoilation after.  Yours seems to only show the errors but not the standard outut.  That try/catch is just a convenience to try ifort first, then if that isn't available to use gfortran.  I can make it less automatic and make people change the makefile manually.

**Comment by andrewning** [comment]
@taylormcd Is that all that you see when you build?  It doesn't show the gfortran commands?

**Comment by taylormcd** [comment]
Yes, that's all I see. It seems the Pkg module is hiding those commands.  I see the gfortran commands if I run the build script manually.
